functions/deleteShow.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+const AWS = require('aws-sdk');
+const httpResponse = require('../httpResponse')
+
+
+module.exports.handle = async (event) => {
+    const id = event.pathParameters.id
```

Comment:
Pode usar o spread pra pegar valores dentro de objetos
```suggestion
    const { id } = event.pathParameters
```